src/flow/styled.js

```diff
@@ -0,0 +1,122 @@
+// @flow
+import type {
+  UnconditionalStyles,
+  ConditionalStylesSpec,
+  LiteralStylesSpec,
+  WhitelistedPropNames,
+  ZacsViewFunction,
+  ZacsTextFunction,
+} from './components'
+
+export type Component<T> = React$ComponentType<T>
+
+// NOTE: $Partial is ncessary to make the new props optional. $Shape doesn't do what you think it does
```

Comment:
Wish it was documented somewhere on Flow's website 🙄

Reply:
yeah